Source/WebCore/Modules/encryptedmedia/InitDataRegistry.cpp

```diff
@@ -245,7 +245,7 @@ bool isPlayReadySanitizedInitializationData(const SharedBuffer& buffer)
         return false;
 
     xmlChar* encodedKeyID = xmlNodeGetContent(keyIDNode->nodeTab[0]);
-    std::optional<Vector<uint8_t>> decodedKeyID = base64Decode(encodedKeyID, xmlStrlen(encodedKeyID));
+    std::optional<Vector<uint8_t>> decodedKeyID = base64Decode({ reinterpret_cast<const uint8_t*>(encodedKeyID), static_cast<size_t>(xmlStrlen(encodedKeyID)) });
```

Comment:
The original implementation is:
```
 inline std::optional<Vector<uint8_t>> base64Decode(const void* input, unsigned length, Base64DecodeMode mode)
 {
     return base64Decode({ static_cast<const std::byte*>(input), length }, mode);
 }
```
Why are we changing this to `reinterpret_cast<const uint8_t*>`? It seems `std::span<uint8_t>` would still be casted to `std::span<std::bytes>` later

Reply:
Yeah I remember you are consistently casting char to uint8_t in other places, but this one will be casted to `std::byte` anyways so I wonder whether we should directly convert to std::byte.
```
inline String base64EncodeToString(std::span<const uint8_t> input, Base64EncodeMode mode)
{
    return base64EncodeToString(std::as_bytes(input), mode);
}
```

Reply:
This is not my preference because I am not truing to do std::byte adoption at the moment, just span adoption.